QWeb/keywords/config.py

```diff
@@ -112,8 +112,8 @@ def set_config(par: str, val: Any) -> Any:
     | ScreenShotType_     | Log html source, screenshot or both     | screenshot     |
     +---------------------+-----------------------------------------+----------------+
     | SearchDirection_    | Set relative search direction for       | closest        |
-    |                     | element search (up, down, lef, righ,    |                |
-    |                     | closest)                                |                |
+    |                     | element search (closest, up, down, left,|                |
+    |                     | right, !up, !down, !left, !right)       |                |

```

Comment:
Exclamation marks should be after the direction

Reply:
oops, leftovers when we changed the format. Will update with a new commit